src/main/scala/jigg/util/JSONUtil.scala

```diff
@@ -116,6 +120,64 @@ object JSONUtil {
   }
 
   def writeToJSON[T <: Node](x: T, os: BufferedWriter): Unit = {
-    os.write(toJSON(x).toString)
+    os.write(toJSON(x))
+  }
+
+  def parseJSON(json: String): JValue = parse(json)
+
+  def loadJSONFile(name: String): JValue = {
+    val sb = new StringBuilder
```

Comment:
Full string of the input file can be obtained efficiently by:
```scala
IOUtil.openIterator(name).mkString()
```
`openIterator(filename: String)` returns the iterator of the contents of the file (each element is one line).
`.mkString("\n")` recovers the original file.

It is known that Scala's original IO library has lower quality.